commit 01b01fce2f5a5c04a497a4ad9ee73e1e7737945a
Author: Roberto Oliveira <robertoguimaraes8@gmail.com>
Date:   Mon Apr 2 10:55:37 2018 +0000

    lxd: add missing limits.h include
    
    On systems without glibc, as Alpine, you might lack definition of PATH_MAX.
    This patch adds the limits.h header to solve this issue.
    
    Signed-off-by: Roberto Oliveira <robertoguimaraes8@gmail.com>

diff --git a/lxd/main_forkfile.go b/lxd/main_forkfile.go
index 73f470ca..6c68c896 100644
--- a/lxd/main_forkfile.go
+++ b/lxd/main_forkfile.go
@@ -17,6 +17,7 @@ import (
 #include <string.h>
 #include <sys/stat.h>
 #include <unistd.h>
+#include <limits.h>
 
 extern char* advance_arg(bool required);
 extern void error(char *msg);
